test/smtp.test.lua

```diff
@@ -82,6 +82,29 @@ test:test("smtp.client", function(test)
     m = mails:get()
     test:is_deeply(m.rcpt, {'<cc@tarantool.org>'}, 'no rcpt')
 
+    local ok = pcall(function() return
+    client:request(addr, 'sender@tarantool.org',
+                       nil,
+                       'mail.body',
+                       {})
+                    end)
```

Comment:
The formatting is a bit crazy.

1. `return` on a separate line.
2. The nested code block is on the same indentation level as the parent one.
3. Arguments are indented in some unusual way (I guess due to the unusual indentation of `end`).

I would do this so:

```lua
local ok, err = pcall(function()
    return client:request(addr, 'sender@tarantool.org', nil, 'mail.body')
end)
test:is_deeply({ok, err}, {false, exp_err}, 'No recipients')
```

Or, better:

```lua
local ok, err = pcall(client.request, client, addr, 'sender@tarantool.org',
                      nil, 'mail.body')
```